modules/UpgradeWizard/uw_utils.php

```diff
@@ -4357,58 +4377,6 @@ function add_unified_search_to_custom_modules_vardefs()
     }
 }
 
-/**
- * change from using the older SugarCache in 6.1 and below to the new one in 6.2
- */
-function upgradeSugarCache($file)
```

Comment:
We can deprecate this but we wouldn't be able to completely remove it in a minor release.